src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -93,7 +93,11 @@ public function getHtmlCallGraph()
     public function getProfile()
     {
         if (null === $this->profile) {
-            $this->profile = unserialize($this->data['profile']);
+            $options = null;
+            if (PHP_VERSION_ID >= 70000) {
+                $options = array('allowed_classes' => array('Twig_Profiler_Profile'));
+            }
+            $this->profile = unserialize($this->data['profile'], $options);
```

Comment:
To avoid problems with PHP 5.x this could be changed to:

```php
if (PHP_VERSION_ID >= 70000) {
    $this->profile = unserialize($this->data['profile'], array(
        'allowed_classes' => array('Twig_Profiler_Profile'),
    ));
} else {
    $this->profile = unserialize($this->data['profile']);
}
```